42/README.md

```diff
@@ -126,13 +126,13 @@ election_status=<Election_status>
 
 #### 2. Replay the blockchain
 
-In order to replay the chain up to the migration height, we need to either keep the old Tendermint version running or skip replaying the corresponding part of the chain = archive the chain. We consider the former to be a huge hassle so we conceive an archiving scheme further in this chapter.
+In order to replay the chain up to the migration height, we need to either keep the old Tendermint version running or skip replaying the corresponding part of the chain = archive the chain. We consider the former to be a huge hassle so we describe an archiving scheme further in this BEP.
 
 #### 3. Start the new chain
 
 Validators have to install and launch the new version of Tendermint. They need to prepare a new `genesis.json`. The new genesis file has to contain the validator set, the application hash (both at the migration height), and the identifier of the new Tendermint chain (`chain_id`).
 
-The application hash is calculated by computing the Merkle tree root of the current UTXO.
+The application hash is calculated by computing the Merkle tree root of the current UTXO. Note that hash calculation is a subject to design in a separate BEP and can be changed in the future.
```

Comment:
I don't know understand why you kept a specific method of computing the application hash in this BEP.

Moreover, we don't calculate the application hash like that today, and I'm not convinced we'll compute it that way in the future. There's a lot more to the current BigchainDB state than the UTXO set. The full state includes all `asset.data` and `metadata` information, for example.

Reply:
I used UTXO root since it can be used to do some useful verifications in the future, unlike the current hashing.

It's not indeed important at the moment because we will instruct the users to take the hash and the dump from the same validator - we do not anyway offer tools to verify the dump-hash integrity yet. So I can as well just recommend to use the already implemented way.